packages/design-system-web/src/components/geov-entity-properties/geov-entity-properties.tsx

```diff
@@ -0,0 +1,206 @@
+import { Component, Host, h, Prop, State } from '@stencil/core';
+import { sparqlJson } from '../../lib/sparqlJson';
+
+const qrOntomeURI = (id: string) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT ?object
+WHERE {
+  OPTIONAL{geov:${id} rdf:type ?object .}
+}
+LIMIT 1
+`;
+
+const qrTypeAndLabel = (id: string) => `
```

Comment:
the whole query is not needed, since the rdf:type and rdfs:label can be queried together with the outgoing properties, by making the rdgs:label optional, see below.